moz.build

```diff
@@ -30,6 +30,9 @@ if CONFIG['CPU_ARCH'] == 'x86_64':
             'mac64/install.rdf',
         ]
 
+if CONFIG['OS_TARGET'] == 'Darwin' or CONFIG['OS_TARGET'] == 'Linux':
+    FINAL_TARGET_FILES['chrome']['b2g-installer']['content'] += ['subprocess_worker_unix.js']
```

Comment:
@lissyx because we only want to pack the worker for unix. We don't need the windows version.